examples/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.nii*
```

Comment:
I wonder if we should not be more precise, and use explicitly "*.nii" and "*.nii.gz"